packages/terra-application/src/application-base/ApplicationBase.jsx

```diff
@@ -111,21 +94,34 @@ const ApplicationBase = ({
 
   const { localeOverride, themeOverride } = useTestOverrides(); // Allows us to test deployed applications in different locales.
 
+  const finalLocale = localeOverride || locale || browserLocale;
+
+  useEffect(() => {
+    if (finalLocale !== undefined) {
+      i18nLoader(finalLocale).then(translationsModule => setMessages(translationsModule.default)).catch((error) => {
+        // eslint-disable-next-line no-console
+        console.error(error);
```

Comment:
We should use the terra-application logger as suggested here: https://github.com/cerner/terra-application/pull/124/files#r565468654